app.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Yaydoc",
+  "description": "Automatic Documentation Generation and Deployment Project.",
+  "keywords": [
+    "documentation",
+    "fossasia",
+    "node",
+    "sphinx"
+  ],
+  "repository": "https://github.com/fossasia/yaydoc",
+  "env": {
+    "PYPANDOC_PANDOC": {
```

Comment:
@niranjan94  These are Github's CLIENTID and CLIENTSECRET etc. How will the user be asked to set these value whilst clicking `Deploy to Heroku` button?